src/citrine/resources/data_objects.py

```diff
@@ -151,7 +151,7 @@ def _get_attribute_bounds_search_body(attribute_bounds):
         }
 
     def validate_templates(self,
-                           model: DataObjectResourceType,
+                           model: DataObjectResourceType, *,
```

Comment:
I think we might want to put the asterisk before `model`, since in this case the name `validate_templates` doesn't have a clear, single argument that obviously goes first. And in general, the user will specify a model and some templates.